Providers/Modules/Plugins_x64/ProcessInvestigator/plugin/processinvestigator

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+sessionId=$(cat /proc/sys/kernel/random/uuid)
+if [ $(expr length "$sessionId") -ne 36 ]
+then
+    sessionId="00000000-0000-0000-0000-000000000000"
+fi
+
+piCmd="/opt/microsoft/omsagent/plugin/pi --scanmode=periodic"
+
+# After 10s send SIGTERM, PI should handle that and exit gracefully.
+# If PI is not terminanted 5 seconds after SIGTERM send SIGKILL
+sleep 10 && pkill -f -x "$piCmd" && sleep 5 && pkill -9 -f -x "$piCmd" &
```

Comment:
This line does pkill on a process in /opt/microsoft/omsagent/ 

Doing that sets off an alert because the it looks like someone tampering with the oms agent

ProviderName: MSTIC
AlertType: VM_SuspectProcessTermination

Reply:
Getting quite a few FPs of this over the past few days.